app/konnect/analytics/index.md

```diff
@@ -1,37 +1,61 @@
 
 You can use the following table to help you determine which analytics tools are best for your use case:
 
 | You want to... | Then use... |
 | -------------- | ----------- |
 | Quickly understand API usage and performance at a glance, and gain insights across your entire organization. | [Summary dashboard](/konnect/analytics/dashboard/) |
 | Browse API usage data to gain access to key performance and health statistics. Visualize, slice, and dice API usage data you’ve stored in only a few clicks. | [Explorer](/konnect/analytics/explorer/) |
-| Communicate insights across teams or departments and share important KPIs. | [Custom reports](/konnect/analytics/custom-reports/) |
-| Understand user behavior or pin point where issues occur. See detailed information about each individual request made to your APIs. | [Requests](/konnect/analytics/api-requests/)  |
-
+| Communicate contextual insights across teams or departments and share important KPIs. | [Custom reports](/konnect/analytics/custom-reports/) |
+| Understand user behavior or pinpoint where issues occur. See detailed information about each individual request made to your APIs. | [Requests](/konnect/analytics/api-requests/)  |
+| Manage Advanced Analytics cost for individual control planes| [Scoping control planes](/konnect/analytics/#ingestion) |
 
 ### Charts and limitations
 
-{{site.konnect_short_name}} charts offer interactive features such as the ability to hover over elements to reveal additional details, and to filter data by selecting items with the chart. {{site.konnect_short_name}} caps the number of entities showing in any activity graph or [custom reports](/konnect/analytics/generate-reports/) to 50. If this limit is exceeded, a warning icon displays at the top of the affected graph or report. 
-
+{{site.konnect_short_name}} charts offer interactive features such as the ability to hover over elements to reveal additional details, and to filter data by selecting items with the chart. {{site.konnect_short_name}} caps the number of entities showing in any activity graph or [custom reports](/konnect/analytics/use-cases/) to 50. If this limit is exceeded, a warning icon displays at the top of the affected graph or report.
 
 ### Team permissions
 
 You can assign {{site.konnect_short_name}} users to specific, predefined Analytic teams. This allows you to give certain users access to only view or manage the Analytics area of your {{site.konnect_short_name}} instance. For more information about the Analytics Admin and Analytics Viewer teams, see the [Teams Reference](/konnect/org-management/teams-and-roles/teams-reference/).
 
+
+### Ingestion
+
+The Advanced Analytics toggle feature allows you to control whether Advanced Analytics data is collected and processed for your API traffic through a control plane.
```

Comment:
```suggestion
The Advanced Analytics toggle feature allows you to control whether Advanced Analytics data is collected and processed for your API traffic within a given control plane.
```

Reply:
Given that some customers have expressed confusion about whether data flows through the control plane, I'd like to clarify that the data isn't flowing through the CP, only through DPs within a CP.  I'm not 100% sure if I'm successful here.

Reply:
might also consider:

> API traffic associated with a particular control plane.

or "belonging to a particular control plane"?